Updates/fixes for noodle demo (#73)
tweedegolf/cfg-noodle: demos/README.md

# Noodle Demo

> This is a minimal example application using the `cfg-noodle` crate (`/crates/cfg-noodle`).

Reviewer: ```suggestion
This is a minimal example application using the [`cfg-noodle` crate](./../crates/cfg-noodle).
```

* 1.25hz (400ms on, 400ms off)
* 0.625hz (800ms on, 800ms off)

> Each time the speed is changed via button press, the value is stored back to the node. These changes are then flushed back to the external flash. If you power cycle the device, the LEDs should resume blinking at the same speed they were at power-off. (Unless you power-cycled before the flash-write has completed.)

Reviewer: ```suggestion
Each time the speed is changed via button press, the value is stored back to the node. These changes are then flushed back to the external flash. If you power cycle the device, the LEDs should resume blinking at the same speed they were at power-off, unless power was lost before the flash-write completed.
```
